crates/env/src/backend.rs

```diff
@@ -46,7 +46,7 @@ impl ReturnFlags {
     }
 
     /// Returns the underlying `u32` representation.
-    #[cfg(not(feature = "ink-experimental-engine"))]
+    #[cfg(not(feature = "std"))]
```

Comment:
Why does this need to be behind a feature flag?

Reply:
In `std` the function is never used.